src/org/labkey/test/components/SetDefaultPageClass.java

```diff
@@ -0,0 +1,19 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+
+public class SetDefaultPageClass extends LabKeyPage
+{
+    public SetDefaultPageClass(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public void selectDefaultGrid(String gridName)
+    {
+        click(Locator.linkWithText("select").withAttributeContaining("onclick",gridName));
```

Comment:
This click triggers a page load:
```suggestion
        clickAndWait(Locator.linkWithText("select").withAttributeContaining("onclick",gridName));
```